llvm/test/CodeGen/RISCV/GlobalISel/double-zfa.ll

```diff
@@ -0,0 +1,79 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+
+; RUN: llc -mtriple=riscv32 -mattr=+zfa,d -global-isel < %s \
+; RUN: | FileCheck %s --check-prefixes=CHECK-ZFA-f64
```

Comment:
You can drop --check-prefixes and let it use the default of `CHECK`

Reply:
Dropped additional check prefix